boards/cc1350-launchpad/Makefile.features

```diff
@@ -0,0 +1,8 @@
+CPU = cc26x0_cc13x0
+CPU_MODEL = cc13x0f128
+
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_i2c
```

Comment:
I'm surprised `periph_spi` has not been implemented yet for this family, that tends to be one of the simpler peripherals.

Reply:
This is on my to do list, I gave it a try before to see how would the code would look but didn't finished it

There is the `periph_flashpage` too which I implemented but somehow I managed to brick the MCU on the write operation, and with the erase too by a rookie mistake when erasing the last flash page (which contains the CCFG) locking me out. (See [diff](https://github.com/RIOT-OS/RIOT/compare/master...btcven:2020_12_20-flashpage))